ci/autoscaler/tasks/update-sdk/create_pr.sh

```diff
@@ -44,5 +44,7 @@ pushd "${autoscaler_dir}" > /dev/null
 
   git push --set-upstream origin "${update_branch}"
 
-  gh pr create --base main --head "${update_branch}" --title "${pr_title}" --body "${pr_description}" --label 'approved' --label 'dependencies'
+  gh pr create --base main --head "${update_branch}" --title "${pr_title}" --body "${pr_description}" --label 'approved' --label 'dependencies' --label 'allow-acceptance-tests'
```

Comment:
This wont work. Dependencies will be enough ... although we should remove the approved label.

Reply:
there seems to be another issue in here

Reply:
I believe this is the same issue that we had with dependabot updates because there were 2 labels added in quick succession i.e. approved and dependencies. 
We should only have one label  'dependencies'  and this should make it work.

Reply:
ok we fixed this as per your recomendation